docs/how_to/console/project_development.mdx

```diff
@@ -1,10 +1,31 @@
 ---
-id: project_dev_tools_setup
-title: Project Development Tools Setup
-sidebar_label: Project Development Tools Setup
+id: project_development
+title: Project Development
+sidebar_label: Project Development
 ---
 
-In this guide you will see how to configure a list of development tools in order to use them in the Mia-Platform Console.
+In this guide you will see how to configure:
+- services inside the Mia-Platform Console
+- development tools in order to use them in the Mia-Platform Console.
+
+## Microservice communication intra-namespace
+
+In this guide you will see how microservices can interact and communicate inside the same namespace.
+
+The Mia-Platform Console offers an overview for all microservices created in which is visible a section [Container ports](/development_suite/api-console/api-design/microservice-container-ports.md) where you can view and manage container ports.
+
+![Environment variable section](img/container_ports.png)
+
+Suppose you have two microservices in the same namespace; `service-A` that needs to call `service-B` on the `GET /my-api` endpoint.
```

Comment:
service names in k8s do not accept upper case characters; i'd not make an example with `service-A` but with `service-a`

Reply:
changed